src/System.Management.Automation/resources/SessionStateStrings.resx

```diff
@@ -296,6 +296,9 @@
   </data>
   <data name="ClearContentProviderException" xml:space="preserve">
     <value>Attempting to perform the ClearContent operation on the '{0}' provider failed for path '{1}'. {2}</value>
+  </data>
+    <data name="ClearDirectoryContent" xml:space="preserve">
+    <value>Unable to clear content of '{0}' because it is a directory. Clear-Content cmdlet has to be used on a file.</value>
```

Comment:
For the second sentence, would prefer:

Clear-Content is only supported on files.

Reply:
sure, will correct it.